.github/ISSUE_TEMPLATE/out_of_date_package_report.md

```diff
@@ -0,0 +1,48 @@
+---
+name: Out-of-date package reports
+about: For packages that are out-of-date
+title: ''
+labels: '0.kind: out-of-date package'
+assignees: ''
+
+---
+
+
+###### Checklist
+
+<!-- Note that these are hard requirements -->
+
+<!--
+You can use the "Go to file" functionality on github to find the package
+Then you can go to the history for this package
+Find the latest "package_name: old_version -> new_version" commit
+The "new_version" is the the current version of the package
+-->
+- [ ] Checked the [nixpkgs master branch](https://github.com/NixOS/nixpkgs)
+<!--
+Type the name of your package and try to find an open pull request for the package
+If you find an open pull request, you can review it!
+You will probably have the new version of the package while helping the community!
```

Comment:
s/probably/eventually

Reply:
Oh woaw, I had trouble understanding that it was a substitution command from vim xD
that's smart, I like it!
The suggestion is also smart ;)
I'll take care of it tommorow morning

Reply:
If they know how to run the code in the PR, they'll have it right away!

Reply:
Yeah, I was going for that feeling to push the user to actually review the PR they found!
I'm going to reformulate while I'm at it tho to make it clearer :P

Reply:
Looks good to me now